source/Calamari/Deployment/SpecialVariables.cs

```diff
@@ -106,6 +112,8 @@ public static class Features
         public static class Action
         {
             public const string Name = "Octopus.Action.Name";
+            public const string SkipRemainingConventions = "Octopus.Action.SkipRemainingConventions";
```

Comment:
Should the variables collection be used for passing variables between conventions?  If they are never used outside of the Calamari conventions.  It seems it should be more for passing between server and Tentacle/Calamari.  In the case of SkipRemainingConventions, and SkipJournal etc, we could simply set properties on the RunningDeployment?